fedimint-core/src/bitcoin_migration.rs

```diff
@@ -8,6 +8,66 @@ pub fn bitcoin29_to_bitcoin30_public_key(pk: bitcoin::PublicKey) -> bitcoin30::P
         .expect("Failed to convert bitcoin v29 public key to bitcoin v30 public key")
 }
 
+pub fn bitcoin29_to_bitcoin30_secp256k1_public_key(
```

Comment:
I wish we weren't just duplicating these functions everywhere. But I guess the faster we  merge all these PRs the sooner we can get rid of them.

Reply:
Yeah I decided to hold off on more bitcoin v0.30 PRs until we merge the ones I have out already for this reason. Sorry for the mess! I think we've added most of the commonly needed conversion functions by now.